cmd/start.go

```diff
@@ -154,3 +167,20 @@ func loadStaticKey(
 
 	return privateKey, publicKey, nil
 }
+
+func waitForStake(stakeMonitor chain.StakeMonitor, address string, timeout int) error {
+	waitMins := 0
+	for waitMins < timeout {
+		hasMinimumStake, err := stakeMonitor.HasMinimumStake(address)
+		if err != nil {
+			return fmt.Errorf("could not check the stake [%v]", err)
+		}
+		if hasMinimumStake {
+			return nil
+		}
+		fmt.Printf("%s stake is below required minimum, wait duration: %d minutes \n", address, waitMins)
```

Comment:
My instinct says that this should be a log. Especially with @nkuba investigating the move to ipfs-log v2, which will allow logging to contain structured data, it would probably help surface things better.

Thoughts? Open to sticking to a `Printf` if that seems like overkill.

Reply:
Oh one other note---we do typically wrap at 80 characters. Haven't figured out a great way to handle that on the auto-format front, so I have to apologize for that 😅

Reply:
I would vote to stick with log wherever it's possible. This allows operators to parse the logs in a consistent way.

Reply:
I'm wondering if this message maybe qualifies even to a warn level?

Reply:
I agree, using the logger is better. I've made this change. I've also shortened the log message so the line is under 80 chars.

Reply:
I went with info, thought it made sense. I can change to warn if you want @nkuba .

Reply:
On further thought, I think warn level is better. I've made that change.

Reply:
A lot happened here since last I looked :laughing: